roles/ovirt-host-deploy-ovirt-metrics/tasks/main.yml

```diff
@@ -0,0 +1,4 @@
+---
```

Comment:
Can you please create patch to oVirt engine, on top of: https://gerrit.ovirt.org/#/c/83870/ ?